src/js/components/mainMenu.js

```diff
@@ -29,7 +31,8 @@ class Menu {
     this.menu.classList.add('menu');
     this.menu.innerHTML = `
     ${this.createMenuItem('Новая игра', 'start')}
-    ${this.createMenuItem('Продолжить', 'continue disabled')}
```

Comment:
why did you delete class "disabled"? when user load game at first time, Continue menu item is not able because game not started yet

Reply:
I did it for testing load game